packages/docs/cookbook/testing.md

```diff
@@ -94,23 +94,54 @@ store.name = 'my new name'
 store.$patch({ name: 'new name' })
 expect(store.name).toBe('new name')
 
-// actions are stubbed by default but can be configured by
-// passing an option to `createTestingPinia()`
+// actions are stubbed by default, so this call won't actually
+// execute the implementation defined by the store. See below 
+// to customize this behavior.
 store.someAction()
 
 expect(store.someAction).toHaveBeenCalledTimes(1)
 expect(store.someAction).toHaveBeenLastCalledWith()
 ```
 
-If you are not using Jest (for example, you are using vitest), you will need to provide a [createSpy](https://pinia.vuejs.org/api/interfaces/pinia_testing.testingoptions.html#createspy) to `createTestingPinia()`, like:
+Please note that if you are using Vue 2, `@vue/test-utils` requires a [slightly different configuration](#unit-test-components-vue-2).
+
+
+### Customizing behavior of actions
+
+By default, `createTestingPinia` stubs out all store actions, such that they are not executed when called during unit tests. This allows you to test your components and stores separately.
```

Comment:
```suggestion
`createTestingPinia` stubs out all store actions unless told otherwise. This allows you to test your components and stores separately.
```